app/views/customers/_form.html.erb

```diff
@@ -0,0 +1,36 @@
+<%= simple_form_for(customer,
+                    wrapper: :horizontal_form) do |form| %>
+  <%= form.input :name,
+                 label: false,
+                 placeholder: :template,
+                 required: false %>
```

Comment:
Do you really need the `required` attributes... I assume the customer here is of type `Customer`?